components/Accordion/Item.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="d-flex flex-column accordion-item" :class="{'flex-grow-1': expanded}">
```

Comment:
Do you think we should eventually consider the Vue dynamic class or style syntax an antipattern? We could probably move this kind of thing into computed props eventually, it'll make the longer but more clear.

Reply:
I think it's fine for one- at most two- classes. It gets out of hand when it's multiple lines, and especially when there's a mix of `[]` and `{}` class syntax (like in some components). For just one class, the benefits are questionable.